### DISCUSSION TITLE
BitShares Suez 7.0.2

### DISCUSSION THREAD

**Discussion by abitmore** [bitshares/bitshares-core#2803] [General]
The [BitShares](https://bitshares.github.io) Core software has been updated to the [7.0.2 Bugfix Release](https://github.com/bitshares/bitshares-core/milestone/46?closed=1). The codename of the `7.x` releases is *"Suez"*. The BitShares Core software is used to build the validation nodes that perform consensus of all transactions on the BitShares blockchain. This release includes improvements but _does not include any changes to the consensus protocol_.

Documentation for users may be be found at [How BitShares Works](https://how.bitshares.works).

Documentation for BitShares developers may be found at the [BitShares Developer Portal](https://dev.bitshares.works).

* [Who should upgrade](#who-should-upgrade-7-0-2)
* [How to upgrade](#how-to-upgrade-7-0-2)
* [Binaries for download](#binaries-for-download-7-0-2)
* [Changes](#changes-7-0-2)
* [Release contributors](#release-contributors-7-0-2)

## <div id="who-should-upgrade-7-0-2"/> Who Should Upgrade

**Note: This version is experimental, and every older version has its known issues. Please do your own research to determine which version suits your needs.**

Please check the [release notes of the `7.0.0` version](https://github.com/bitshares/bitshares-core/releases/tag/7.0.0) for more info about the BitShares Suez update.

### Precautions

- Blockchain replay is not needed and will not occur automatically when upgrading to this version.

### How to Upgrade with Docker

The pre-built Docker image can be found on the [BitShares Core Docker page](https://hub.docker.com/r/bitshares/bitshares-core/) and can be downloaded with the following command.

```
docker pull bitshares/bitshares-core:7.0
```

### <div id="upgrade-from-source-7-0-2" /> How to Upgrade from Source Code

**Note:**

* The code base requires a compiler that supports the C++ 14 standard
* The minimum supported boost version is 1.58, the maximum is 1.74
* The minimum supported CMake version is 3.2
* For Ubuntu users, it is recommended to build in Ubuntu 20.04 LTS (Focal Fossa).
  * Ubuntu 18.04 LTS (Bionic Beaver) and earlier versions are no longer supported.
  * Ubuntu 22.04 LTS (Jammy Jellyfish) is not yet supported.

#### Obtain the Source Code

The source code may be obtained by checking out the [7.0.2 tag](https://github.com/bitshares/bitshares-core/releases/tag/7.0.2).  Download the source at:

* [7.0.2.zip](https://github.com/bitshares/bitshares-core/archive/7.0.2.zip)
* [7.0.2.tar.gz](https://github.com/bitshares/bitshares-core/archive/7.0.2.tar.gz)

#### Build the Binaries

The binaries may be built by using your pre-existing process, or by following the standard instructions that can be found in the wiki:

* [Ubuntu](https://github.com/bitshares/bitshares-core/wiki/BUILD_UBUNTU)
* [macOS](https://github.com/bitshares/bitshares-core/wiki/Building-on-macOS)
* [Windows](https://github.com/bitshares/bitshares-core/wiki/BUILD_WIN32)

#### Deploy the Binaries

Your standard process for deploying the node software may be used. No additional requirements or precautions will be required to deploy the new release.

# <div id="binaries-for-download-7-0-2"/> Binaries for download

Binaries of the command-line interface (CLI) wallet and the node software are pre-built and available for download.

Note:
* These binary builds are reproducible and were built using [Gitian](https://github.com/bitshares/bitshares-gitian/tree/7.0.2). Please check the [README of Gitian](https://github.com/bitshares/bitshares-gitian/blob/master/README.md) if you want to verify and/or contribute.
* Starting from version `7.0.0`, pre-built Linux binaries will not run on Ubuntu 18.04 LTS (Bionic).

|Platform|SHA256 Checksum|
|-|-|
|[Microsoft Windows](https://github.com/bitshares/bitshares-core/releases/download/7.0.2/bitshares-core-7.0.2-win64-bin.zip)|`6d13c3597cbffe1950fbe18e04f243b15ce3f36ec47c858c67cc7db4ee26762a`|
|[Mac](https://github.com/bitshares/bitshares-core/releases/download/7.0.2/bitshares-core-7.0.2-macos64-bin.tar.bz2)|`099d5f3b4068f2d95ec52bb6946987d6fb6897be8afbd608f520878918d249b8`|
|[Linux](https://github.com/bitshares/bitshares-core/releases/download/7.0.2/bitshares-core-7.0.2-linux-amd64-bin.tar.bz2)|`f3e8f61cac53db5ef9d3ce6776445ca1b5c4156d91f498fdc11d8691d471bc81`|

# <div id="changes-7-0-2"/> Changes

The changes for [7.0.2 Bugfix Release](https://github.com/bitshares/bitshares-core/milestone/46?closed=1) are summarized below.

## <div id="node-performance-7-0-2">Node Performance and Stability</div>

|Description|Issue|Pull Request|
|-|-|-|
|Try to fix a sync issue|[2798](https://github.com/bitshares/bitshares-core/issues/2798)|[2799](https://github.com/bitshares/bitshares-core/pull/2799)|

# <div id="release-contributors-7-0-2"/> Release Contributors

@abitmore (https://github.com/abitmore)

<hr /><em>This discussion was created from the release <a href='https://github.com/bitshares/bitshares-core/releases/tag/7.0.2'>BitShares Suez 7.0.2</a>.</em>

**Comment by olaf0523** [comment]
hi there
